Connections/canvas-context-connections.json

```diff
@@ -62,17 +62,167 @@
       "connectionDescription": "Belongs to the same endpoint",
       "parentContextKey": "Endpoint"
     },
+
     {
-      "contextKey1": "Hostname",
-      "contextKey2": "MAC",
-      "connectionDescription": "Belongs to the same endpoint",
-      "parentContextKey": "Endpoint"
+      "contextKey1": "SSDeep",
+      "contextKey2": "MD5",
+      "connectionDescription": "Belongs to the same file",
+      "parentContextKey": "File"
     },
     {
-      "contextKey1": "MAC",
+      "contextKey1": "Username",
+      "contextKey2": "Email.Address",
+      "connectionDescription": "User's email address",
+      "parentContextKey": "Account"
+    },
+    {
+      "contextKey1": "Username",
+      "contextKey2": "Domain",
+      "connectionDescription": "User's domain",
+      "parentContextKey": "Account"
+    },
+    {
+      "contextKey1": "Email.Domain",
+      "contextKey2": "Email.Address",
+      "connectionDescription": "Email's domain",
+      "parentContextKey": "Account"
+    },
+
+    {
+      "contextKey1": "Hostname",
       "contextKey2": "IP",
       "connectionDescription": "Belongs to the same endpoint",
       "parentContextKey": "Endpoint"
+    },
+
+    {
+      "contextKey1": "Hash",
+      "contextKey2": "RelatedDomains",
+      "connectionDescription": "Related domain",
+      "parentContextKey": "File"
+    },
+    {
+      "contextKey1": "Hash",
+      "contextKey2": "RelatedHosts",
+      "connectionDescription": "Related host",
+      "parentContextKey": "File"
+    },
+    {
+      "contextKey1": "Hash",
+      "contextKey2": "RelatedIPs",
+      "connectionDescription": "Related ip",
+      "parentContextKey": "File"
+    },
+    {
+      "contextKey1": "Hash",
+      "contextKey2": "RelatedURLs",
+      "connectionDescription": "Related url",
+      "parentContextKey": "File"
+    },
+    {
+      "contextKey1": "Hash",
+      "contextKey2": "CommunicatedHosts",
+      "connectionDescription": "Communicated with host",
+      "parentContextKey": "File"
+    },
+    {
+      "contextKey1": "Hash",
+      "contextKey2": "CommunicatedIPs",
+      "connectionDescription": "Communicated with host",
```

Comment:
@erezh31 shouldn't this be "Communicated with IP"? As down below you refer to "CommunicatedDomains" as "Communicated with domain"